packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -158,10 +161,13 @@ const App: React.FC<AppProps> = observer(
               onDragStarted={() => setIsResizing(true)}
               onDragFinished={() => setIsResizing(false)}
               onChange={onSplitPaneChange}
+              // For some reason on each dom snapshot restoring the viewport is jumping up to 4 pixels
+              // It causes a weird white line in the bottom, so here is a fix which is not ideal
+              paneStyle={{ height: 'calc(100vh + 4px)' }}
               className={cs('reporter-pane', { 'is-reporter-resizing': isResizing })}
             >
-              <div>
-                {state.spec && (
+              <div style={{ height: '100%' }}>
```

Comment:
A utility class like `full-height` will be perfect here, but I don't like global styles :) so instead of making another `specs-list-runner-container` inlining the style makes it more clear what we need right here